data/Mitchell_2008/metadata.yml

```diff
@@ -0,0 +1,521 @@
+    2004 and February-March 2005 collected between 0400 and 0700 hours. Four or five
+    leaf or leafy twig samples per species were rehydrated and transported to the
+    laboratory, and curves were generated using the bench-drying technique (Turner
+    1988). The two leaf physiological parameters included in this study represent
+    the seasonal shifts in these traits calculated using the difference in summer
+    and spring values, and denoted by delta-psi-P100 and delta-Emax.
+- var_in: Demax
+  unit_in: MPa
+  trait_name: .na.character
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 4-5
+  methods: Leaf physiological parameters related to turgor maintenance, namely osmotic
+    potential (Psi-P100) and maximum bulk modulus of elasticity (Emax) during spring
+    and summer, were taken from related work involving 20 of the 21 species (Mitchell
+    et al. 2008). Pressure-volume analyses were performed on leafy twigs in September
+    2004 and February-March 2005 collected between 0400 and 0700 hours. Four or five
+    leaf or leafy twig samples per species were rehydrated and transported to the
+    laboratory, and curves were generated using the bench-drying technique (Turner
+    1988). The two leaf physiological parameters included in this study represent
+    the seasonal shifts in these traits calculated using the difference in summer
+    and spring values, and denoted by delta-psi-P100 and delta-Emax.
+- var_in: KL-NO Ks
+  unit_in: 10^5.kg/m/MPa/s
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 6-8
+  methods: All stems for hydraulic conductivity measurements were collected in the
+    field between 0400 and 0700 hours to minimise the effect of diurnal changes in
+    hydraulic conductivity (K). Six to eight stems per species were sampled during
+    September 2004 and March 2005. Twig samples were collected from different individuals
+    during each season to minimise the impact of the sampling on the small plants
+    of many of the species. Large leafy branches were cut (distal to the leaves) and
+    bagged in large plastic bags containing moist paper towels, sealed and re-bagged,
+    transported to the laboratory and stored at 4ºC. At the time of measurement of
+    K, stems were re-cut under water at a length approximately two times greater than
+    the maximum vessel length (0.2-0.8 m), and stem widths varying from 3 to 10 mm.
+    The apparatus for measuring hydraulic conductivity was similar to that described
+    by Sperry et al. (1988). (See paper for more details). Leaf area was also measured
+    on leaves above the proximal end of the stem segments for determination of leaf-specific
+    hydraulic conductivity (KL).
+  measurement_remarks: leaf_specific_conductivity, but data in Supplementary Information
+    are for Ks.
+- var_in: conduit diam
+  unit_in: mm
```

Comment:
How did you figure out the units was mm here? If it were cm it would still be within the allowable range.